ocs_ci/templates/workloads/helper_scripts/fsync.py

```diff
@@ -0,0 +1,16 @@
+import os
+
+
+def fsync():
```

Comment:
Please explain here what this function do. This is the core function to the automation of this bug.